tools/runners/run-all.js

```diff
@@ -163,6 +178,8 @@ class Runner {
 
     if (! self.stopped) {
       buildmessage.enterJob({ title: "starting your app" }, function () {
+        self.errorAppRunner.start();
+        runLog.log('Started error app.', {arrow: true});
```

Comment:
I wonder if the users even need to see this line. I.e. they probably don't know what the error app is. As opposed to other things like "starting mongo" and "starting your app". Although, I would agree that the current "starting proxy" is not super helpful either.